addons/mail/data/mail_groups.xml

```diff
@@ -1,9 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 <data noupdate="1">
+    <record model="ir.module.category" id="module_category_canned_response">
+        <field name="name">Canned Responses</field>
+    </record>
+
     <record id="group_mail_canned_response_admin" model="res.groups">
         <field name="name">Canned Response Administrator</field>
-        <field name="category_id" ref="base.module_category_hidden"/>
+        <field name="category_id" ref="module_category_canned_response"/>
```

Comment:
This record is is inside `noupdate`, so it will require an upgrade script to update it for existing db

Reply:
https://github.com/odoo/upgrade/pull/6141